MvvmCross/iOS/iOS/Platform/MvxIosSetup.cs

```diff
@@ -42,10 +42,11 @@ protected MvxIosSetup(IMvxApplicationDelegate applicationDelegate, UIWindow wind
             _applicationDelegate = applicationDelegate;
         }
 
-        protected MvxIosSetup(IMvxApplicationDelegate applicationDelegate, IMvxIosViewPresenter presenter)
+        protected MvxIosSetup(IMvxApplicationDelegate applicationDelegate, UIWindow window, IMvxIosViewPresenter presenter) : this (applicationDelegate, window)
         {
-            _presenter = presenter;
+            _window = window;
             _applicationDelegate = applicationDelegate;
```

Comment:
`_applicationDelegate` and  `_window` are already set in the call for `: this (applicationDelegate, window)`, no need to assigning them again here